Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM tomcat:8.5-alpine
```

Comment:
We've been using Tomcat 9 in images.  Use the `tomcat:9-jdk11-openjdk-slim` image as a base.

Reply:
See comment below about removing the Dockerfile for now.